app/presenters/tslr_claims_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class TslrClaimsPresenter < BasePresenter
+  attr_accessor :decorated_collection
+
+  def initialize(claims)
+    self.decorated_collection = claims.map { |v| TslrClaimPresenter.new(v) }
+    super(claims)
+  end
+
+  def csv_headers
+    TslrClaim.csv_fields.map { |h| translation_for_field(h) }
+  end
+
+  private
+
+  def translation_for_field(header)
```

Comment:
How about `header_string_for_field` or something so that it says what it does, not how it does it.